libtransmission/rpcimpl.cc

```diff
@@ -957,7 +955,21 @@ static char const* setLabels(tr_torrent* tor, tr_variant* list)
             return "labels cannot contain comma (,) character";
         }
 
-        labels.emplace(label);
+        labels->emplace(label);
+    }
+
+    return nullptr;
+}
+
+static char const* setLabels(tr_torrent* tor, tr_variant* list)
+{
+    auto labels = tr_labels_t{};
+    char const* errmsg = nullptr;
+
+    errmsg = makeLabels(&labels, list);
```

Comment:
_(nit)_ these can be joined into a single line that reduces the scope of errmsg:

```c++
if (auto const* const errmsg = makeLabels(&labels, list); errmsg != nullptr)
{
    return errmsg;
}
```